### ISSUE TITLE
Clean up keyword n_avg

### ISSUE THREAD

**Issue by rlbyrne** [EoRImaging/FHD#197]
The keyword `n_avg` need a default reported in the dictionary. It seems to be used in both gridding and Healpix outputs and has different defaults in different parts of the code. We need to clean up, document, and possibly rename the keyword.

**Comment by bhazelton** [comment]
Agreed. I find it particularly confusing that there is another keyword, `nfreq_avg` which means something totally different but sounds very similar every time I see it.

**Comment by mwilensky768** [comment]
@bhazelton not that it solves this issue, but I'm working on a PR that just changes `nfreq_avg` to `beam_nfreq_avg` which is slightly more illustrative.

**Comment by bhazelton** [comment]
That will help a lot, thanks!
